dashboard/app/views/reports/assume_identity_form.html.haml

```diff
@@ -1,14 +1,14 @@
 %h1 Assume identity 
 
-- if flash[:errors]
```

Comment:
oh wait, actually... instead of doing all of this you could just use flash[:notice] and it will be taken care of by the show_flashes helper that is called in the layout:

https://github.com/code-dot-org/code-dot-org/blob/a1260471a7aa0a8ebdfdcd9e026614e84ec13898/dashboard/app/views/layouts/application.html.haml#L29

https://github.com/code-dot-org/code-dot-org/blob/a1260471a7aa0a8ebdfdcd9e026614e84ec13898/dashboard/app/helpers/application_helper.rb#L77